smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/ErrorHandlerIT.java

```diff
@@ -48,22 +67,34 @@ public static void beforeClass() {
     }
 
     /**
-     * Ensure the author returns a 404 when requesting a non-exising page.
+     * Ensure the author returns a 404 when requesting a random non-exising page.
      *
      * @throws ClientException
      */
     @Test
     public void testAuthorResponseCode404() throws ClientException {
-        adminAuthor.doGet("/content/does/not/exist.html", 404);
+        String path = "/content/test-site/" + UUID.randomUUID();
```

Comment:
```suggestion
        String path = "/content/test-site/missing-" + UUID.randomUUID();
```

There's a very small chance, but the random uuid could hit a page that exists :)